src/index.html

```diff
@@ -9,6 +9,57 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
```

Comment:
Remember about empty lines between parent and child elements.
```suggestion
    <header class="header">
```